.github/ISSUE_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+## Issue that this pull request solves

```

Comment:
@Aman-Codes this comes under #46 , which is being fixed under #47 
Get rid of these changes. Rest seems fine :+1: